01-2024-schedule/course-schedule/week-4.md

```diff
@@ -7,3 +7,66 @@ description: Second-Layer Applications
 This week, we'll mainly learn about Second-Layer Applications, specifically the Lightning Network. We'll explore how the network functions, its structure, and payment channels.&#x20;
 
 Additionally, we'll delve into topics like path finding and how payments are delivered in the Lightning Network. Plus, we'll try to grasp the philosophy behind Bitcoin.
+
+## Prerequisites
+
+* LND should be installed and ready
+* Developer environment from week 1 should be available. Having [Polar](https://lightningpolar.com) installed and functional is essential! It requires [Docker](https://www.docker.com/products/docker-desktop/) to be installed.
+
+## Goals
+
+* Understand how the lightning network operates
+  * Why it is thought of as a bitcoin "layer 2"
+  * Where the trust in the system lies
+  * What security assumptions it makes
+  * How the various "layers" of the network fit together
+* Understand the architecture of Lightning Network
+* Lightning Nodes and Wallets
+* Be familiar with sending and receiving lightning payments
+* Understand Payment Channels
+  * Constructing a Payment Channel
+  * Sending Payment accross the Channel
+  * The Commitment Transaction
+  * Advancing the Channel State
+  * Closing the Channel
+* Understand Routing on a network of Payment Channels
+  * Routing a Payment
+  * Routing vs pathfinding
+  * Creating a network of Payment Channels
+  * Fairness Protocol
+  * Hash Time-Locked Contracts (HTLCs)
+* Understand Channel Operation and Payment Forwarding
+  * Managing HTLCs
+  * Committing HTLCs to channel state
+  * Channel balance
+* Understand Path Finding and Payment Delivery
+  * Finding Candidate Paths
+  * Payment Delivery
+  * Multiparts Payment (MPP)
```

Comment:
Can this be Multiparts/Multipath Payment(MPP)? They are used interchangeably and might be confusing when searching for external resources.

Reply:
Thank you for the observation. However, in the `lnbook`, it is stated that `the term multipart payment is more accurate than multipath payment` Check Multipart Payment (MPP) in the [glossary](https://lnbook.256k1.dev/#glossary).   AMP (Atomic Multipath Payment) is used more often when referring to Multipath.